edenia/antelope-tools
# Feat/flag disable missed blocks 1216

### Add flag to disable missed blocks services

### What does this PR do?

- Resolve #1216 

### Steps to test

1. Run the project locally 
2. Check that the missed block service is not executed
